### DISCUSSION TITLE
react-instantsearch-hooks and naturalLanguages

### DISCUSSION THREAD

**Discussion by ottob** [algolia/instantsearch#5687] [Q&A] [answered]
Hello, im using
https://www.algolia.com/doc/guides/building-search-ui/going-further/native/react-hooks/

and now I want to try natural languages queries. I have found https://www.algolia.com/doc/api-reference/api-parameters/naturalLanguages/ but I can not figure out how to pass this option into the hooks. Does anyone know? Should it go into the `<InstantSearch` component somehow?

**Comment by Haroenv** [comment, marked as answer]
Search parameters that aren't associated to a particular widget can be passed as props to the Configure widget: https://www.algolia.com/doc/api-reference/widgets/configure/react-hooks/. In this case that would be `<Configure naturalLanguages={["en"]} />`

**Reply by ottob** [reply]
Great, thanks! For anyone else looking to use it with react-native: `Configure` can also be imported from `react-instantsearch-hooks`.
